Tests/SentryTests/Integrations/SentryCrash/SentryCrashReportTests.swift

```diff
@@ -169,7 +171,26 @@ class SentryCrashReportTests: XCTestCase {
         XCTAssertNil(mach.code_name)
         XCTAssertEqual(0, mach.subcode)
     }
-    
+
+#if os(iOS) && !targetEnvironment(macCatalyst)
+    // We can't really test writing the crash_info_message unless there is an actual crash.
+    // We noticed that the libsystem_sim_platform.dylib is a simulator library that has a crash info message even when there is no crash.
+    // This is simply a smoke test to ensure we can write the crash_info_message key to the report.
+    // This only works on iOS and also not macCatalyst.
+    // Actual testing must be done manually with a fatalError in Swift.
+    func testWriteCrashReport_ContainsCrashInfoMessage() throws {
```

Comment:
`l`: Do you think there could be value in creating a workflow/test-runner-system which actually starts the sample app in the simulator in CI and let it crash, to then look at the data written to disk outside of Xcode? Maybe using something like NPM jest as the test runner, executing shell scripts like we do in the sentry-wizard E2E tests

Reply:
That should be `h` 😄. Yes indeed, but that's not straightforward. I opened an issue: https://github.com/getsentry/sentry-cocoa/issues/5096.